Automattic/jetpack
# PHPCS: Fix WordPress.Security.ValidatedSanitizedInput outside of Jetpack and WAF

#### Changes proposed in this Pull Request:

Mostly this means adding `wp_unslash()` and `filter_var()` calls, and
sometimes `isset()` checks too. Occasionally it means ignoring the sniff
in code that's already doing better than `filter_var()` would do.

Jetpack is skipped for now because it's big. WAF is skipped for now
because it's not immediately clear to me whether WordPress's
`wp_unslash()` is loaded at the time it runs.

#### Jetpack product discussion


Followup to #23932

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy?
* Double check that it actually ran the "PHP Code Sniffer (non-excluded files only)" job.
* Check that the validation seems correct. Note in particular that `filter_var()` will turn an array to false, if that's what happened to be passed.
